### ISSUE TITLE
support /my/directory/<anything.yml> for the github-actions ecosystem

### ISSUE THREAD

**Issue by staabm** [dependabot/dependabot-core#5970]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature description

when configuring a custom directory for the github-actions ecosystem, it seems dependabot is searching for a `action.yml` and does not allow me to put any other named file into this directory.

see the error in the screenshot:
![grafik](https://user-images.githubusercontent.com/120441/197717593-a058f485-f43d-4f1b-9465-0032c8e5a330.png)

**Actual:**
dependabot is searching only for `/templates/workflows/action.yml` workflow files

**Expected:**
dependabot should search for `/templates/workflows/*.yml` workflow files, like it is doing in the default path `/.github/workflows/<anything>.yml.`

----

I have a repo in which I use the following dependabot config:

```
version: 2
updates:
  - package-ecosystem: "github-actions"
    directory: "/templates/workflows/"
    schedule:
      interval: "daily"
```

the projects file-system structure is

![grafik](https://user-images.githubusercontent.com/120441/197717341-96549509-1471-432c-8a42-26ad60cf1a8c.png)

**Comment by martinm82** [comment]
I have the same issue. In my case I have bunch of workflows defined in a cookiecutter template folder (see the duplicate I created #5987 )

**Comment by deivid-rodriguez** [comment]
This was fixed by #6189 🎉

**Comment by staabm** [comment]
can confirm that today morning we got dependabot PRs in such workflows. thats awesome - thank you
